packages/gatsby/cache-dir/static-entry.js

```diff
@@ -5,13 +5,9 @@ import { kebabCase, get, merge, isArray, isString } from "lodash"
 import apiRunner from "./api-runner-ssr"
 import pages from "./pages.json"
 import syncRequires from "./sync-requires"
+import requireWithFallback from './require-with-fallback'
 
-let Html
-try {
-  Html = require(`../src/html`)
-} catch (e) {
-  Html = require(`./default-html`)
-}
+const Html = requireWithFallback(`../src/html`, `./default-html`)
```

Comment:
for any require in `cache-dir` yes, you can rescope the helper to contain the check and throw if you want though. but you need to do the require with string literals